src/main/java/frc/robot/Constants.java

```diff
@@ -228,16 +234,28 @@ public static class VisionConstants {
     public static final String TURRET_CAMERA_NAME = "Camera_Module_Turret";
   }
 
+  public static class ClimberConstants {
+    public static final int LEFT_ARM_ID = 20;
+    public static final int RIGHT_ARM_ID = 11;
+    public static final double CLIMB_PERCENT = -0.4;
+    public static final double CLIMB_POSITION = 7;
+    public static final double[] POSITION_PID = new double[] {0.01, 0, 0};
+    public static final double[] CURRENT_PID = new double[] {0.01, 0, 0};
+    public static final double MIN_RAISE_TIME_SEC = 0.1;
+    public static final double CLIP_CURRENT = -10;
+  }
+
   public static class enabledSubsystems {
     public static final boolean intakeEnabled = true;
     public static final boolean drivetrainEnabled = true;
-    public static final boolean visionEnabled = true;
-    public static final boolean usingPhoton = true;
+    public static final boolean visionEnabled = false;
```

Comment:
Let's turn these on to true when testing next.  Vision should be plugged in, and the limelight box will be used to stream the driver camera